lib/pay.rb

```diff
@@ -3,6 +3,7 @@
 require "pay/errors"
 require "pay/adapter"
 
+require "action_mailer"
 require "active_support/dependencies"
```

Comment:
Not related to this PR exactly, but I wonder if we even need this?

Want to try removing it and see if all the tests pass?

Reply:
Yeah, that sounds good. I'll update now.